devices.js

```diff
@@ -1490,6 +1490,13 @@ const devices = [
         description: 'B22 Bulb RGBW',
         extend: generic.light_onoff_brightness_colortemp_colorxy,
     },
+    {
+        zigbeeModel: ['RB 250 C'],
+        model: 'RB 250 C',
+        vendor: 'Innr',
+        description: 'E27 Bulb RGBW',
```

Comment:
Isn't this a E14 bulb? (https://www.coolblue.nl/product/831417/innr-colour-e14-rb-250-c.html)

Reply:
oh sry yes.... its e14

Reply:
corr plz merge